internal/cmd/iam/command_groupmapping_delete.go

```diff
@@ -0,0 +1,53 @@
+package iam
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+)
+
+func (c *groupMappingCommand) newDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "delete <id>",
+		Short:             "Delete a group mapping.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE:              c.delete,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Delete group mapping "pool-12345":`,
+				Code: "confluent iam group-mapping delete pool-12345",
```

Comment:
I thought the "pool" prefix was for identity pools?

Reply:
It is, and it's planned to be changed to `group` in Q4. I think for the LA release in September it will still be `pool`. Would it be better to change it to `group` now or retroactively update it when the API is updated?

Reply:
Let's keep it as `pool`, I guess. Can you open a ticket for this so we don't forget?